src/app/pages/home/helpCard.js

```diff
@@ -1,26 +1,34 @@
-import { Button, Card, Title } from "@newfold/ui-component-library";
-import SupportIllustration from "../../images/section-home-help-me.svg";
-export function HelpCard() {
-  return (
-    <Card className="wppbh-help-card">
-      <Card.Content>
-        <Title size={2}>Need some help?</Title>
-        <div className="nfd-flex nfd-flex-col nfd-gap-4 nfd-items-center nfd-text-center nfd-mt-4">
-          <img 
-            className="nfd-max-w-xs" 
-            src={SupportIllustration} 
-            alt="Help Agent Illustration"
-          />
-          <p>From DIY to full-service help.</p>
-          <p>
-            Call or chat 24/7 for support or let our experts build your site for
-            you.
-          </p>
-          <Button variant="secondary" as="a" href="#/help" className="wppbh-help-link">
-            Help me
-          </Button>
-        </div>
-      </Card.Content>
-    </Card>
-  );
-}
+import { Button, Card, Title } from '@newfold/ui-component-library';
+import SupportIllustration from 'App/images/section-home-help-me.svg';
+
+const HelpCard = ( {} ) => {
+	return (
+		<Card className="wppbh-help-card">
+			<Card.Content>
+				<Title size={ 2 }>Need some help?</Title>
+				<div className="nfd-flex nfd-flex-col nfd-gap-4 nfd-items-center nfd-text-center nfd-mt-4">
+					<img
+						className="nfd-max-w-xs"
+						src={ SupportIllustration }
+						alt="Help Agent Illustration"
+					/>
+					<p>From DIY to full-service help.</p>
```

Comment:
:1234: Need to make this translatable

Reply:
Addressed in this commit: https://github.com/bluehost/bluehost-wordpress-plugin/pull/820/commits/f2189f1882d8bc9b38edf951a32c2bcc2d63133f